app/src/test/java/com/example/marvelapp/SplashViewModelTest.kt

```diff
@@ -1,7 +1,7 @@
 package com.example.marvelapp
```

Comment:
the ViewModel tests should be inside a ViewModel package

Reply:
done